alias/cuda/activation1d.py

```diff
@@ -3,19 +3,26 @@
 
 import torch
 import torch.nn as nn
-from alias_free_torch.resample import UpSample1d, DownSample1d
+from torch.resample import UpSample1d, DownSample1d
```

Comment:
I think having the name of the anti-aliased activation module as `alias_free_activation` may be better (just `alias` might not look clear to the users at the first sight). Since this has been one of the key method of BigVGAN, I'd like to make the name of the module clearer.

Minor, but having the torch vs. cuda impl as `torch` and `cuda` might confuse the users (assuming these method originate from the native pytorch, for example).

Can you run the CUDA kernel auto-build using `use_cuda_kernel=True` during the model init and confirm that it works like before this refactor?